crates/sncast/src/starknet_commands/deploy.rs

```diff
@@ -50,6 +49,18 @@ pub struct Deploy {
     pub rpc: RpcArgs,
 }
 
+#[derive(Debug, Clone, clap::Args)]
+#[group(multiple = false)]
+pub struct DeployArguments {
+    /// Arguments of the called function serialized as a series of felts
+    #[clap(short, long, value_delimiter = ' ', num_args = 1..)]
+    pub constructor_calldata: Option<Vec<String>>,
```

Comment:
Is this name the only reason that we need a separate struct instead of using `Arguments`?

Reply:
Yes, it's only the name. Don't want to break this, it was called `constructor_calldata` previously.

Reply:
It seems reasonable to me to make this breaking change and reuse the `Arguments` struct here. Maybe in a different PR and for a future release (not today's) but generally I think it should be done.